[Appliance Upgrade] Add instructions to ensure domain and licence matches during 10.17 to 10.18 upgrade

raghuvamsi-d opened the pull request.

raghuvamsi-d committed:
Add instruction to keep domain and licence same during migraiton

bbyreddy commented on content/edge/edge-update-bundle/migration-1017-to-1018.md:
```diff
@@ -56,6 +56,8 @@ Install the Edge appliance 10.18 following the instructions below.
+After installing Edge 10.18 appliance, configure the Edge domain and license to match those of the older Edge Appliance VM you are migrating. For detailed steps, refer to [Configuring Edge] (/edge/edge-configuration)
 {{< /c8y-admon-important >}}
 
 1. Refer to [Configuring the Edge infrastructure](/edge/edge-infrastructure/) for detailed instructions on importing the Edge 10.18 appliance.
```
A note here would be ideal, stating that they should use the Edge domain and license same as that of the Edge 10.17 instance.

bbyreddy commented on content/edge/edge-update-bundle/migration-1017-to-1018.md:
```diff
@@ -56,6 +56,8 @@ Install the Edge appliance 10.18 following the instructions below.
 {{< c8y-admon-important >}}
 If you have both the Edge 10.17 and Edge 10.18 appliances on the same host machine, ensure that the IP addresses of both appliances are different.
+
+After installing Edge 10.18 appliance, configure the Edge domain and license to match those of the older Edge Appliance VM you are migrating. For detailed steps, refer to [Configuring Edge] (/edge/edge-configuration)
```
Install the Edge 10.18 appliance with the domain and license to match those of the older Edge Appliance VM you are migrating.

raghuvamsi-d committed:
Revise installation instructions for Edge 10.18

Updated instructions for installing Edge 10.18 appliance to ensure correct domain and license configuration.

BeateRixen commented on content/edge/edge-update-bundle/migration-1017-to-1018.md:
```diff
@@ -58,6 +58,10 @@ Install the Edge appliance 10.18 following the instructions below.
 {{< /c8y-admon-important >}}
 
+{{< c8y-admon-info >}}
+Install the Edge 10.18 appliance with the domain and license to match those of the older Edge Appliance VM you are migrating. For detailed steps, refer to [Installing Edge] (/edge/edge-installation)
```
```suggestion
Install the Edge 10.18 appliance with the domain and license to match those of the older Edge Appliance VM you are migrating. For detailed steps, refer to [Installing Edge](/edge/edge-installation).
```

BeateRixen committed:
Update content/edge/edge-update-bundle/migration-1017-to-1018.md

BeateRixen commented on content/edge/edge-update-bundle/migration-1017-to-1018.md:
```diff
@@ -58,6 +58,10 @@ Install the Edge appliance 10.18 following the instructions below.
 {{< /c8y-admon-important >}}
 
+{{< c8y-admon-info >}}
+Install the Edge 10.18 appliance with the domain and license to match those of the older Edge Appliance VM you are migrating. For detailed steps, refer to [Installing Edge](/edge/edge-installation).
```
```suggestion
Install the Edge 10.18 appliance with the same domain and license as those of the older Edge Appliance VM you are migrating. For detailed steps, refer to [Installing Edge](/edge/edge-installation).
```

raghuvamsi-d committed:
Update content/edge/edge-update-bundle/migration-1017-to-1018.md

Co-authored-by: Beate Rixen <90445236+BeateRixen@users.noreply.github.com>
